hack/update-deps.sh

```diff
@@ -18,49 +18,10 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-export GO111MODULE=on
+source $(dirname "$0")/../vendor/knative.dev/test-infra/scripts/library.sh
 
-source $(dirname $0)/../vendor/knative.dev/test-infra/scripts/library.sh
+go_update_deps "$@"
 
-cd ${REPO_ROOT_DIR}
-
-VERSION="master"
-
-# The list of dependencies that we track at HEAD and periodically
-# float forward in this repository.
-FLOATING_DEPS=(
-  "knative.dev/pkg@${VERSION}"
-  "knative.dev/eventing@${VERSION}"
-  "knative.dev/serving@${VERSION}"
-  "knative.dev/test-infra@${VERSION}"
-)
-
-# Parse flags to determine any we should pass to dep.
-GO_GET=0
-while [[ $# -ne 0 ]]; do
-  parameter=$1
-  case ${parameter} in
-    --upgrade) GO_GET=1 ;;
-    *) abort "unknown option ${parameter}" ;;
-  esac
-  shift
-done
-readonly GO_GET
-
-if (( GO_GET )); then
-  go get -d ${FLOATING_DEPS[@]}
-fi
-
-# Prune modules.
-go mod tidy
-go mod vendor
-
-rm -rf $(find vendor/ -name 'OWNERS')
-rm -rf $(find vendor/ -name 'OWNERS_ALIASES')
-rm -rf $(find vendor/ -name 'BUILD')
-rm -rf $(find vendor/ -name 'BUILD.bazel')
-rm -rf $(find vendor/ -name '*_test.go')
-
-export GOFLAGS=-mod=vendor
-
-update_licenses third_party/VENDOR-LICENSE "./..."
+# Patch k8s leader-election fixing graceful release
+# More information: https://github.com/kubernetes/kubernetes/pull/91942
+git apply ${REPO_ROOT_DIR}/hack/k8s-client-go.patch
```

Comment:
do you need this patch? you did not use it before.

Reply:
we applied it to eventing-kafka-broker too https://github.com/knative-sandbox/eventing-kafka-broker/pull/293

Reply:
based on feeback, I removed that for now

Thanks @n3wscott